multi_fetch_controller/ViewPoint view_point
---
bool is_success
